bramkouwen/Neatly-In-Motion
# Creating Website CSS

## What does this PR do?
* It creates main.css 
* It creates the code for main.css

## How do you check this PR?
* Pull down the branch
* Check the added main.css file  
* Check the code

## Screenshots 
None.
